crypto/libtomcrypt/Kconfig

```diff
@@ -0,0 +1,57 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig CRYPTO_LIBTOMCRYPT
+	bool "LibTomCrypt CrypographyLibrary"
+	default n
+	select MATH_LIBTOMMATH
+	---help---
+		LibTomCrypt is a fairly comprehensive, modular and portable
+		cryptographic toolkit that provides developers with a vast array
+		of well known published block ciphers, one-way hash functions,
+		chaining modes, pseudo-random number generators, public key
+		cryptography and a plethora of other routines. 
+
+if CRYPTO_LIBTOMCRYPT
+
+config LIBTOMCRYPT_VERSION
+	string "LibTomCrypt Version"
+	default "1.18.2"
+
+menuconfig LIBTOMCRYPT_DEMOS
+	bool "LibTomCrypt Library Demos"
+	default n
+	---help---
+		LibTomCrypt demo and test applications.
+
+if LIBTOMCRYPT_DEMOS
```

Comment:
can we remove LIBTOMCRYPT_DEMOS? LIBTOMCRYPT_LTCRYPT is enough.

Reply:
I believe @btashton let it in a menuconfig in case the other demos from https://github.com/libtom/libtomcrypt/tree/develop/demos may be included in the future.